README.md

```diff
@@ -252,6 +252,31 @@ Example Playbook
     - wireguard
 ```
 
+Run tests
+---------
+
+Make sure to have vagrant installed.
```

Comment:
Just installing vagrant isn't enough - at least not for Archlinux. It should be also mentioned that this either also needs `VirtualBox` installed (as this is the default here) or the Vagrant libvirt plugin (as described in the Archlinx wiki e.g. https://wiki.archlinux.org/index.php/Vagrant#vagrant-libvirt). At least we should make people aware that not only Vagrant needs to be installed but also a suitable provider like `VirtualBox` or `libvirt` plugin.

Reply:
done